maintenance/scripts/setup.sh

```diff
@@ -39,41 +39,51 @@ ${parent_path}/update_api_env.sh
 chown -R www-data ./storage ./vendor
 chmod -R 775 ./storage
 
-# setup common project
-cd /var/www/html/common
+# setup frontend workspace
+cd /var/www/html/frontend
+# Part of hydrogen work-around. See below.
+rm -rf */node_modules/@hydrogen-design-system
 nvm install --latest-npm
 npm install
+npm rebuild node-sass
+
+# Work-around until hydrogen is updated: We trick hydrogen into thinking its installed in the package path.
+# See: https://github.com/hydrogen-design-system/hydrogen.css/issues/72
+mkdir -p admin/node_modules common/node_modules talentsearch/node_modules
+(cd admin/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+(cd common/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+(cd talentsearch/node_modules && ln -s ../../node_modules/@hydrogen-design-system .)
+# Another Hydogren work-around for executable bit not being enabled, leading to issues.
+# See: https://github.com/hydrogen-design-system/hydrogen.css/issues/73
+chmod +x node_modules/@hydrogen-design-system/hydrogen.css/bin/*
```

Comment:
Temporary fix until this has time for a release: https://github.com/hydrogen-design-system/hydrogen.css/issues/73